compiler/rustc_mir_transform/src/abort_unwinding_calls.rs

```diff
@@ -54,11 +55,11 @@ impl<'tcx> MirPass<'tcx> for AbortUnwindingCalls {
         };
         let body_can_unwind = layout::fn_can_unwind(tcx, Some(def_id), body_abi);
 
-        // Look in this function body for any basic blocks which are terminated
-        // with a function call, and whose function we're calling may unwind.
+        // Look in this function body for any basic blocks which end with a
+        // terminator that has a target (e.g. function call) which may unwind.
         // This will filter to functions with `extern "C-unwind"` ABIs, for
         // example.
-        let mut calls_to_terminate = Vec::new();
```

Comment:
I personally prefer the old naming, it was clearer: it's fine to "call" inline assembly since it's treated as a function call in most stages of the compiler all the way down to LLVM. "Target operation" is pretty vague to the point of being confusing.